bootstrap/src/basis/map_intf.ml

```diff
@@ -137,6 +137,17 @@ module type S = sig
       comparator and the [veq] value comparison function, [false] otherwise.
       O(n) time complexity. *)
 
+  val subset: ('v -> 'v -> bool) -> of_:(('k, 'v, 'cmp) t) -> ('k, 'v, 'cmp) t
```

Comment:
I'm wondering if the `~of_` named parameter is a design mistake.  Consider that `diff`'s parameter order matters, but there we don't name either parameter.  It seems to me the same design considerations hold here.

Reply:
I was reading the `of_` version this morning and actually had to think about it a bit. The `t0 t1` was immediately a bit more obvious. Probably because most of the api looks more like the latter.